src/main/java/com/softserve/academy/spaced/repetition/repository/CardRatingRepository.java

```diff
@@ -7,8 +7,8 @@
 
 public interface CardRatingRepository extends JpaRepository<CardRating, Long> {
 
-    @Query("select avg(c.rating) from CardRating c where c.cardId=:cardId group by c.cardId")
-    Double findRatingByCardId(@Param("cardId") long cardId);
+    @Query("select avg(c.rating) from CardRating c where c.card.id=:cardId group by c.card.id")
+    Double findRatingByCard_Id(@Param("cardId") long cardId);
```

Comment:
Why is this underline needed here?

Reply:
Because here we have refence key.

Reply:
What does this underline mean? It's just a symbol, can it change something?

Reply:
When we have foreign key, according to the arrangements we should name method so.
It is just convention.